src/Storage/Storage/DatalakeGen2/Cmdlet/NewAzDataLakeGen2Item.cs

```diff
@@ -83,9 +83,10 @@ public string Source
         [ValidatePattern("([r-][w-][x-]){3}")]
         public string Umask { get; set; }
 
-        [Parameter(Mandatory = false, HelpMessage = "Sets POSIX access permissions for the file owner, the file owning group, and others. Each class may be granted read, write, or execute permission. Symbolic (rwxrw-rw-) is supported. ")]
+        [Parameter(Mandatory = false, HelpMessage = "Sets POSIX access permissions for the file owner, the file owning group, and others. Each class may be granted read, write, or execute permission. Symbolic (rwxrw-rw-) is supported. " +
+            "The sticky bit is also supported and its represented either by the letter t or T in the final character-place depending on whether the execution bit for the others category is set or unset respectively, absence of t or T indicates sticky bit not set.")]
         [ValidateNotNullOrEmpty]
-        [ValidatePattern("([r-][w-][x-]){3}")]
+        [ValidatePattern("([r-][w-][x-]){2}[r-][w-][xtT-][+]{0,1}")]
```

Comment:
I just test the suffix "+" looks not work on server.
And it's also not in our rest API, so we might should not add it now.
Would you please help to revert it by remover "[+]{0,1}"?

Reply:
Done